src/Evolution/Systems/Cce/Initialize/InitializeJ.hpp

```diff
@@ -80,28 +81,86 @@ struct GaugeAdjustInitialJ {
 };
 
 /// \cond
-struct InverseCubic;
 struct NoIncomingRadiation;
 struct ZeroNonSmooth;
+template <bool uses_inertial_coordinates>
+struct InverseCubic;
+template <bool uses_inertial_coordinates>
+struct InitializeJ;
```

Comment:
[minor]  I know this is just a forward declaration, but maybe rename the template param here?